storybook/webpack.config.js

```diff
@@ -58,10 +48,11 @@ module.exports = (baseConfig, env, defaultConfig) => {
     }
   );
 
-  defaultConfig.resolve.alias = {
-    ...defaultConfig.resolve.alias,
+  config.resolve.alias = {
+    ...config.resolve.alias,
     'patternfly-react': path.resolve(__dirname, '../packages/patternfly-3/patternfly-react/src')
   };
 
-  return defaultConfig;
+  console.log('config', config.module.rules)
```

Comment:
Do we need to log this ... can it be removed.

Reply:
Good catch, thanks.